app/controllers/SiteController.php

```diff
@@ -120,11 +120,13 @@ public function getGraphData(){
   public function lrs(){
     $opts = ['user' => \Auth::user()];
     $lrss = $this->lrs->index($opts);
-    $counts = $this->lrs->getCounts();
+    $lrs_repo = $this->lrs;
 
+    $collection = \DB::getCollection('statements');
 
-    return Response::json(array_map(function ($lrs) use ($counts) {
-      $lrs->statement_total = isset($counts[$lrs->_id]) ? $counts[$lrs->_id] : 0;
+    return Response::json(array_map(function ($lrs) use ($lrs_repo) {      
+      $query = ['lrs_id'=>new \MongoId($lrs->_id)];
```

Comment:
Yea you're right, I moved the count to the repo method and the query didn't get removed